plugins/rapture/nexus-coreui_legacy-plugin/src/main/resources/static/rapture/NX/coreui_legacy/controller/Repositories.js

```diff
@@ -112,7 +112,7 @@ Ext.define('NX.coreui_legacy.controller.Repositories', {
           beforedestroy: me.stopStatusPolling
         },
         'nx-coreui_legacy-repository-feature button[action=browse]': {
-          click: me.navigateToBrowseMode
+          runaction: me.navigateToBrowseMode
```

Comment:
This change seems both counter-intuitive and not consistently applied: why are we moving from a 'known' event like click to runaction and why only for certain buttons?